admin_guide/manage_rbac.adoc

```diff
@@ -710,60 +701,58 @@ Subjects:
 
 ifdef::openshift-enterprise,openshift-origin[]
 [[creating-local-role]]
-== Creating a Local Role
+== Creating a local role
 
-To create a local role for a project, run the following command:
+You can create a local role for a project and then bind it to a user.
 
+. To create a local role for a project, run the following command:
++
 ----
-$ oc create role ...
+$ oc create role <name> --verb=<verb> --resource=<resource> -n <project>
 ----
-
-The following excerpt from the help of this command describes its usage:
-
++
+In this command, specify:
+* `<name>`, the local role's name
+* `<verb>`, the verbs to apply to the role
+* `<resource>`, the resources that the role applies to
+* `<project>`, the project name
++
+For example, to create a local role that allows a user to view pods in the
+`blue` project, run the following command:
++
+----
+$ oc create role podview --verb=get --resource=pod -n blue
 ----
-Create a role with single rule.
-
-Usage:
-  oc create role NAME --verb=verb --resource=resource.group/subresource [--resource-name=resourcename] [--dry-run] [options]
-
-Examples:
-  # Create a Role named "pod-reader" that allows user to perform "get", "watch" and "list" on pods
-  oc create role pod-reader --verb=get --verb=list --verb=watch --resource=pods
-
-  # Create a Role named "pod-reader" with ResourceName specified
-  oc create role pod-reader --verb=get,list,watch --resource=pods --resource-name=readablepod --resource-name=anotherpod
-
-  # Create a Role named "foo" with API Group specified
-  oc create role foo --verb=get,list,watch --resource=rs.extensions
-
-  # Create a Role named "foo" with SubResource specified
-  oc create role foo --verb=get,list,watch --resource=pods,pods/status
 
-Options:
-      --dry-run=false: If true, only print the object that would be sent, without sending it.
-      --resource=[]: resource that the rule applies to
-      --resource-name=[]: resource in the white list that the rule applies to, repeat this flag for multiple items
-      --verb=[]: verb that applies to the resources contained in the rule
+. To bind the new role to a user, run the following command:
```

Comment:
@kalexand-rh Looks much better without the help text. If you wanted to be industrious, you could convert them to properly-formatted examples.

Reply:
@mburke5678, I thought about making a table of example commands, but I didn't think they were different enough! Thank you for rechecking this one for me. :)